dev/com.ibm.ws.cdi.third.party_fat/fat/src/com/ibm/ws/cdi/thirdparty/tests/HibernateSearchTest.java

```diff
@@ -96,11 +100,17 @@ public static void setUp() throws Exception {
             LibertyServerFactory.getLibertyServer(SERVER_NAME);
         }
 
+        //On AIX we can't just use /tmp (see https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.workitem.viewWorkItem&id=300059 and the links in comment one)
+        //So we load the persistence.xml file, and give it a path from the server.
```

Comment:
```suggestion
        //We used to use /tmp here, but we can't do that on some platforms which we build and test on
        //So we load the persistence.xml file, and give it a path from the server.
```
I don't think it's helpful to link to an internal defect in the code - it isn't accessible to anyone external and it'll go stale. Just include the relevant information in the comment. Someone can still track back through the PR if they really need to find more information on the problem.